File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.cs
using System;
using System.Collections.Generic;

public class Solution {
    public int solution(string my_string, string is_suffix) {
        int answer = 0;
        List<string> list = new List<string>();
        
        for(int i = 0; i < my_string.Length; i++)
        {
            list.Add(my_string.Substring(i));
        }
        list.ToArray();
        
        if(list.Contains(is_suffix)) answer = 1;
        else answer = 0;
        return answer;
    }
}

File: 프로그래머스/0/181908. 접미사인지 확인하기/접미사인지 확인하기.cpp
#include <string>
#include <vector>
#include <bits/stdc++.h>

using namespace std;

int solution(string my_string, string is_suffix) {
    vector<string> v;
    for(int i = 0; i < my_string.size(); i++)
    {
        v.push_back(my_string.substr(i));
    }
    if(find(v.begin(), v.end(), is_suffix) != v.end()) return 1;
    else return 0;
}